src/components/Login/LoginErrorAccountNotValidated.js

```diff
@@ -1,46 +1,45 @@
 import React, { useState } from 'react';
 import { useCaptcha } from '../form-helpers/captcha-utils';
-import { injectIntl, FormattedHTMLMessage } from 'react-intl';
+import { useIntl, FormattedHTMLMessage } from 'react-intl';
 import { InjectAppServices } from '../../services/pure-di';
 
-export const LoginErrorAccountNotValidated = injectIntl(
-  InjectAppServices(
-    /**
-     * @param { Object } props
-     * @param { string } props.email
-     * @param { import('react-intl').InjectedIntl } props.intl
-     * @param { import('../../services/pure-di').AppServices } props.dependencies
-     */
-    ({ intl, email, dependencies: { dopplerLegacyClient } }) => {
-      const [resentTimes, setResentTimes] = useState(0);
-      const [Captcha, verifyCaptcha] = useCaptcha();
+export const LoginErrorAccountNotValidated = InjectAppServices(
+  /**
+   * @param { Object } props
+   * @param { string } props.email
+   * @param { import('react-intl').InjectedIntl } props.intl
```

Comment:
This parameter does not exist anymore, right?

Reply:
yes! i removed comment now